content/docs/user-guide/gems/reference/aws/aws-gamelift/cpp-api.md

```diff
@@ -112,10 +111,10 @@ Disconnects the player from the game session.
 To leave the game session, call `AWSGameLiftClientManager::LeaveSession()` or `AWSGameLiftClientManager::LeaveSessionAsync()`.
 
 ```cpp
-// Make synchronous call
+// For example, make synchronous call to leave current session
 AWSGameLift::AWSGameLiftSessionRequestBus::Broadcast(&AWSGameLift::AWSGameLiftSessionRequestBus::Events::LeaveSession);
 
-// Make asynchronous call
+// For example, make asynchronous call to leave current session and get notification after process is complete
```

Comment:
nit: leave the current session and get a notification after the process is complete. 

I would also be wary of "process" here as GameLift has definition of process. Maybe "and get a notification, once the leaving the session is completed"?